src/en/space-station-14/departments/medical/proposals/brigmedic.md

```diff
@@ -0,0 +1,55 @@
+
+*[May 6, 2023](https://github.com/space-wizards/space-station-14/pull/16196):* A PR to revert the brigmedic removal is proposed, citing the majority reactions against the removal. The request is rejected, giving the following reason:
+
+> […] we're not readding the free hardsuit role. The game already has one of those (Paramedic) and it already causes issues. This is the same role except with security access to boot with none of the responsibilities of a medic, and it's primary use seemed to be security with a hardsuit, not an actual medic.
+
+> In other words: Gimmie a well reasoned, thought out design doc for this role, what it should do, what it should not do and how it makes sure those things don't happen, and how it should influence the round, and I'll let you readd it.
+
+This proposal aims to accomplish the goals as requested:
+* What brigmedics should do
+* What brigmedics should not do
+* How does the game make sure those things don't happen
+* How brigmedics should influence the round.
+
+Some thoughts on how to go about fixing the issues are also provided in the PR's thread, which will will be incorporated into this proposal.
+
+*[Jun 4, 2023](https://github.com/space-wizards/space-station-14/pull/17037):* The removal is partially reverted again, bringing back the starting gear loadout for admin use.
+
+## Fixing the problems
+
+> ### This is the same role except with security access to boot with none of the responsibilities of a medic, and it's primary use seemed to be security with a hardsuit, not an actual medic.
+
+We should start off by changing the mindset of brigmedic to focus more on medic instead of brig; it should be brig**medic**, not **brig**medic. Brigmedic should be removed from the security department, and be considered a member of the medical department instead. The brigmedic would still have access to the brig and security channels, similar to the civillian department's lawyer, but just like the lawyer is not a true part of security (i.e., they should not have lethals or cuffs outside of emergencies).
+
+I additionally propose that we treat "brigmedicloosing" as similar to "wardenloosing." The brigmedic *should* always be in the brig, as their title says they should; they are not secmedics. Command should be encouraged to demote brigmedics who abandon their role by leaving the brig. The sole exception to this should be when the brigmedic is instead fufilling the role of a combat medic, in which case they should be in the backlines of sec providing first aid until the officers are well enough to be brought back to the frontlines or a paramedic can retrieve them to bring to medical. We should heavily discourage brigmedics from engaging in combat except as a last resort - they should be filling the same role as a Nuclear Agent does in NukeOps for their team in the combat phase.
```

Comment:
You say they aren't secmedics, but in the next 3/4ths of the paragraph you give precise examples when their role turns into being a secmedic. I do agree that they should stay in the brig though barring these circumstances. They shouldn't be running around making arrests (although in low pop hours, I can see this happening all day every day, and this will be normal when people start allowing it to happen).

The main issue is brigmedics finding every way to abandon their post in the brig and buddy up. These issues should be listed and a possible way to avoid it cited.